hooks.py

```diff
@@ -77,6 +77,23 @@ def get_motd():
             g.MOTD = f.read()
 
 
+@app.before_request
+def get_event():
+    if request.endpoint in ('dynamic_player_image', 'dynamic_server_image'):
+        return
+
+    g.EVENT = None
+
+    event = Variable.get_value('event')
+
+    if event:
+        g.EVENT = {
+            'name': event.name,
+            'datetime': arrow.get(event.datetime),
+            'server': rwr.scraper.get_server_by_ip_and_port(event.server_ip_and_port) if event.server_ip_and_port else None
+        }
```

Comment:
Do not set the event if the current datetime is greater or equal to the event datetime +3hours